codechecker_common/cli.py

```diff
@@ -57,14 +57,25 @@ def get_data_files_dir_path():
     # If it's not an internal package (pypi package), try to find the data
     # directory.
     import sysconfig
-    data_dir_paths = set(sysconfig.get_path("data", s)
-                         for s in sysconfig.get_scheme_names())
+    data_dir_paths = [
+        # Try to find the data directory beside the lib directory.
+        # E.g.: /usr/local/lib/python3.8/dist-packages/codechecker_common/*.py
+        os.path.abspath(os.path.join(__file__, *[os.path.pardir] * 5)),
```

Comment:
This wants to go 5 directories up the tree, right?

Reply:
Yes, exactly. So if the `__file__`=`/usr/local/lib/python3.8/dist-packages/codechecker_common/cli.py` it will be `/usr/local`.

Reply:
Alright. Let's hope it not breaks too much in the future.